src/constants/index.ts

```diff
@@ -1,7 +1,7 @@
 import contractNetworks from '@cowprotocol/contracts/networks.json'
 import { SupportedChainId as ChainId } from './chains'
 
-const { GPv2Settlement } = contractNetworks
+const { GPv2Settlement } = JSON.parse(contractNetworks as unknown as string) as typeof contractNetworks
```

Comment:
this is a hackish solution until rollup (which microbundle uses) add support for import assertions https://github.com/rollup/rollup/issues/3799

having that supported we could do:

```js
import contractNetworks from '@cowprotocol/contracts/networks.json' assert { type: 'json' }
```

Reply:
Wow, that's so hacking that's even funny 😆